src/FitImage.tsx

```diff
@@ -25,23 +72,28 @@ const styles = StyleSheet.create({
   },
 });
 
-class FitImage extends Image {
-  constructor(props) {
+class FitImage extends Component<IFitImageProps, IFitImageState> {
+  public static propTypes = propTypes;
+  private style: ImageStyle;
+  private isFirstLoad: boolean;
+  private mounted: boolean;
+
+  constructor(props: IFitImageProps) {
     super(props);
 
     this.style = StyleSheet.flatten(props.style);
 
     if (this.style) {
       const size = [this.style.width, this.style.height];
 
-      if (size.filter(e => e).length === 1) {
+      if (size.filter((e) => e).length === 1) {
```

Comment:
I figure out `size.filter(Boolean).length === 1` is more readable.

(Yeah, I wrote. 😁)